requirements-dev.txt

```diff
@@ -33,7 +33,7 @@ tifffile; platform_system == "Linux" or platform_system == "Darwin"
 pandas
 requests
 einops
-transformers<4.22; python_version <= '3.10'  # https://github.com/Project-MONAI/MONAI/issues/5157
+transformers>4.36.0
```

Comment:
I think 4.36.0 doesn't have the security vulns, So it can be larger or equal to `>=`